pkg/config/protocol/instance.go

```diff
@@ -137,3 +137,7 @@ func (i Instance) IsGRPC() bool {
 		return false
 	}
 }
+
+func (i Instance) IsUnsupported() bool {
+	return i == Unsupported
```

Comment:
Where does the 'Unsupported' comes from ? Shouldn't it be 'unknown' or 'auto' ? Obviously it is supported - either as TCP or auto.